packages/api-cardano-db-hasura/test/transactions.query.test.ts

```diff
@@ -41,8 +43,40 @@ describe('transactions', () => {
     expect(result.data.transactions[7].blockIndex).toBe(7)
   })
 
+  it('returns an empty array when the transactions has no outputs', async () => {
+    const result = await mc4Client.query({
+      query: gql`query orderedTransactionsInBlock(
+          $hash: Hash32HexString!
+      ) {
+          transactions(
+              where: { hash: { _eq: $hash } },
+          ) {
+              outputs {
+                  address
+                  value
+              }
+              outputs_aggregate {
+                  aggregate {
+                      count
+                  }
+              }
+              inputs_aggregate {
+                  aggregate {
+                      count
+                  }
+              }
+          }
+      }`,
+      variables: { hash: '05e5bb869d0a71e3689d7a7d2bcde70494e370df0e8958fb338562d94a7dddb6' }
```

Comment:
Until we have a local network to cover the e2e, this tx was used as manually verified to not include outputs:
https://github.com/input-output-hk/cardano-explorer-app/issues/329